packages/lesswrong/components/common/SectionFooter.tsx

```diff
@@ -6,7 +6,6 @@ const styles = (theme) => ({
   root: {
     display: "flex",
     justifyContent: "flex-end",
-    alignItems: "center",
```

Comment:
Seems like a risky change, but I checked a couple obvious places (the front page, the user page) and it didn't seem to change anything. So, probably fine.